src/Bundle/Maker/MakeFactory.php

```diff
@@ -218,4 +248,31 @@ private function defaultPropertiesFor(string $class, bool $allFields): iterable
             yield $property['fieldName'] => $value;
         }
     }
+
+    private function hasFactory($factory)
```

Comment:
As discussed in slack, let's use the injected factories instead. `$entitiesWithFactories` should be an array map of `Entity FQCN: Factory FQCN`

Reply:
I'm not sure what you mean. Are you trying to actually run the maker to test the changes?